src/gmail.js

```diff
@@ -1205,6 +1205,11 @@ var Gmail = function(localJQuery) {
         return res;
     };
 
+    api.tools.parse_fd_bv_is_draft = function(item) {
+        if (!Array.isArray(item)) return null;
```

Comment:
We want our `is_draft`-check to return a `boolean`. it should be either `true` or `false`.

Don't return `null` here when you can return `false` instead.

Reply:
The reason I returned `null` is that the item not being an array means the schema has changed (a field that used to always be array-like is no longer populated with array data).  In this case, we really don't know if the message as a draft here, so it seemed nicer to say `null` (not calculable / know) rather than `false` (known to be not a draft). Anyway, if you find this argument unconvincing I don't mind changing!

Reply:
Nobody is going to check for null. Making it false is more consistent with how we otherwise do things. 

Right now the typescript definitions also say this field is Boolean only, so let’s make it so 😉